docs/editor/workspaces.md

```diff
@@ -18,25 +18,25 @@ The concept of a workspace enables VS Code to:
+The easiest way to open a workspace is using the **File** menu and selecting one of the available **Workspace** related entries for opening, such as **Open Workspace...** or **Open Folder...**. Alternatively you might also operate VS Code from a terminal and are able to just pass the path to a workspace as the first argument to the `code` command for opening.
 
 ## Single-folder workspaces
 
-You don't have to do anything for a folder to become a VS Code workspace other than open the folder with VS Code. Once a folder has been opened, VS Code will automatically keep track of things such as your open files and editor layout so the editor will be as you left it when you reopen that folder. You can also add other folder-specific configurations such as workspace-specific [settings](/docs/getstarted/settings.md) (versus global user settings) and [task definition](/docs/editor/tasks.md) and [debugging launch](/docs/editor/debugging.md) files (see below in the [workspace settings](#workspace-settings) section).
+You don't have to do anything for a folder to become a VS Code workspace other than open the folder with VS Code. Once a folder has been opened, VS Code will automatically keep track of things such as your open files and editor layout so the editor will be as you left it when you reopen that folder. You can also add other folder-specific configurations such as workspace-specific [settings](/docs/getstarted/settings.md) (versus global user settings), [task definitions](/docs/editor/tasks.md) and [debugging launch](/docs/editor/debugging.md) files (see below in the [workspace settings](#workspace-settings) section).
 
 ![Single-folder workspace](images/workspaces/single-folder-workspace.png)
 
 *A single-folder workspace opened inside VS Code*
 
 ## Multi-root workspaces
 
-[Multi-root workspaces](/docs/editor/multi-root-workspaces.md) are an advanced capability of VS Code that allow you to configure multiple distinct folders to be part of the workspace. Instead of opening a folder as workspace, you will open a `<name>.code-workspace` JSON file that lists the folders of the workspace. For example:
+[Multi-root workspaces](/docs/editor/multi-root-workspaces.md) are an advanced capability of VS Code that allows you to configure multiple arbitrary folders to be part of the workspace. You can choose which folders to include in a multi-root workspace using the **Add Folder to Workspace...** command in the **File** menu. You can also use the **Add Folder to Workspace...** and **Remove Folder From Workspace** commands of the context menu on folders within the multi-root-workspace in the File Explorer. If you prefer to edit the JSON file, you can also manipulate the set of folders in a multi-root-workspace by opening a `<name>.code-workspace` file that lists the folders of the workspace. For example:
```

Comment:
This topic was meant explicitly to explain what a "Workspace" is and not explain how to use multi root workspaces. I feel we should not be too verbose here because we have an entire topic for multi root workspaces here: https://code.visualstudio.com/docs/editor/multi-root-workspaces